### ISSUE TITLE
How to setup rpc

### ISSUE THREAD

**Issue by tianna1121** [valiant1x/intense-pool#4]
Hi, all

My ITNS pool found first block 4 hours ago. However, the reward cannot sent to miners in my pool. I think the problem is rpc service.

I started my daemon as follows:
./intensecoind

 Maybe I should add some parameters? Thank you!

**Comment by panthar1** [comment]
Seem to have a similar/same problem configuring this;
./walletd -w intensePool.wallet -p my_pass --bind-address 0.0.0.0 --bind-port 48671 --daemon-port 46782

That launches and works. However, on the /admin.html#monitoring page, I see under "wallet" {"code":-32601,"message":"Method not found"}

And from within walletd I see:
2018-Jan-10 10:38:12.445198 WARNING Requested method not found: getbalance

This tells me its connecting to the walletd, but getbalance is for some reason not implemented. I don't know if this will effect payouts on the pool or not?  I have to assume it would.

I  compiled intensecoin from source, so it's up to date as of yesterday. I saw your linux binaries were out of date. Any one have any ideas?

**Comment by panthar1** [comment]
Yeah, just to confirm from my previous message, payments are not working to the workers, same issue as tianna1121 reported. Also, when I restarted the nodejs instance, it lost all the pending payment information, as well. Is there a way to prevent it from losing all this information every time the server is restarted?

**Comment by valiant1x** [comment]
pending payments are stored in `redis`, not node, so your redis config must not be properly dumping or restoring the db

you need to start the wallet separately, and you must pass the `--rpc-bind-port` arg to simplewallet or it cannot interpret the RPC requests from the pool.
